migration/data/course_tables.sql

```diff
@@ -540,29 +541,29 @@ CREATE TABLE forum_posts_history (
 );
 
 CREATE TABLE "thread_categories" (
-	"thread_id" int NOT NULL,
-	"category_id" int NOT NULL
+    "thread_id" int NOT NULL,
```

Comment:
Probably should remove these whitespace changes since they aren't directly related to the PR